README.md

```diff
@@ -1,2 +1,51 @@
-# docker
-GeoServer docker image
+# Docker GeoServer image
+
+This docker GeoServer image is based on the following proposal:
+ 
+https://github.com/geoserver/geoserver/wiki/GSIP-192
+
+Work is still in progress!
+
+## How it works
+
+1. The [Dockerfile](Dockerfile)
+    1. installs dependencies
+    1. installs the GeoServer by downloading and extracting the war file
+    1. installs the marlin renderer
+    1. defines defaults for environment variables
+1. The [entrypoint.sh](scripts/entrypoint.sh) startup script (in a running container)
+    1. executes [install-extensions.sh](scripts/install-extensions.sh) to download and install GeoServer extensions based on the `STABLE_EXTENSIONS` environment variable.
+    1. handles the `GEOSERVER_OPTS`
+    1. starts the tomcat
+
+## Building
+
+`docker build -t geoserver:test .`
+
+## Running
+
+`docker run -it -e STABLE_EXTENSIONS='wps,csw' -p 8080:8080 geoserver:test`
+
+The extensions will be downloaded on startup of the image (before starting the tomcat).
+
+## Configuration
+
+Pass as environment variables. If not passed, the default values will be used.
+
+* `GEOSERVER_DATA_DIR` (default: /opt/geoserver_data)
+* `INITIAL_MEMORY` (default: 2G)
+* `MAXIMUM_MEMORY` (default: 4G)
+* `JAIEXT_ENABLED` (default: true)
+* `STABLE_EXTENSIONS` as comma separated list, will be downloaded and installed on startup (default: "")
+
+## TODOs
+
+* CORS
+* configuration of JNDI connections in the tomcat/custom tomcat configuration in general
+* default data for gs datadir?
+* possibility to add custom java dependencies (for example the oracle jdbc driver)
+  * in this context: mount `EXTENSION_DOWNLOAD_DIR` and implement checks to cache extensions that have already been downloaded before?
+* log4j properties
```

Comment:
please set up a logging profile that does not spam standard out (`stdout`) like most of the defaults do

Reply:
So you are recommending PRODUCTION.properties to start out with?

Reply:
No, because `PRODUCTION.properties` still logs to `stdout` spamming the catalina logs / catalina.out /console with all it thinks we should know about: 

https://github.com/geoserver/geoserver/blob/d09be5b2ebfe0593418a9ce04668be0b17ec3d61/src/main/src/main/java/PRODUCTION_LOGGING.properties#L18

I'd prefer something like `PRODUCTION.properties` with `stdout` removed, so the catalina logs will just tell me things about the servlet engine and not geoserver (because that's what the geoserver log is for)

Reply:
I agree
Maybe we propose to change profile? I don't think it makes sense for production setups